scripts/install_radar_deps.py

```diff
@@ -86,144 +85,147 @@ def get_distribution():
-                "libusb-1.0-0-dev",
-
-                "dpdk-dev",
-                "dpdk-devel",
-
-                "kernel-devel",
-                "linux-headers-generic",
-
-                "unison",
-                ]
-
-    pip = ["deepdish",
-           "posix_ipc",
-           "inotify",
-           "matplotlib",
-           "virtualenv",
-           "protobuf==3.19.4",
-           "numpy",
-           "zmq"]
+    print('### Installing system packages ###')
+
+    common_packages = [
+        "gcc",
+        "gcc-c++",
+        "git",
+        "scons",                # For building Borealis
+        "gdb",                  # For engineeringdebug mode
+        "jq",                   # For reading JSON files from the command line
+        "at",                   # Required for the scheduler
+        "mutt",                 # Required for the scheduler
+        "hdf5",
+        "autoconf",             # Required for installing protobuf and zmq
+        "automake",             # Required for installing protobuf and zmq
+        "libtool",              # Required for installing protobuf and zmq
+        "python3-mako",         # Required for UHD
+        "cmake",
+        "pps-tools",
+        ]
+
+    variant_packages = [
+        "libusb-1_0-{}",    # Needed for UHD
+        "libx11-{}",
+        "pps-tools-{}",
+        "net-snmp-{}",
+    ]
+
+    ubuntu_packages = [
+        "libsnmp-dev",
+        "libusb-1.0-0-dev",
+        "libhdf5-dev",
+        "liboost-all-dev",
+        "libboost-dev",
+        "hdf5-tools",
+        "linux-headers-generic",
+        "libboost-*67.0*",  # Ubuntu 19.10
+        "libboost-*71.0*",  # Ubuntu 20.04
+    ]
+
+    opensuse_packages = [
+        "kernel-devel",
+        "libboost_*_66_0",
+        "uhd-devel",
+    ]
 
     if "openSUSE" in distro:
         pck_mgr = 'zypper'
+        variant_packages = [pck.format('devel') for pck in variant_packages]
+        all_packages = common_packages + variant_packages + opensuse_packages
     elif 'Ubuntu' in distro:
         pck_mgr = 'apt-get'
+        variant_packages = [pck.format('dev') for pck in variant_packages]
+        all_packages = common_packages + variant_packages + ubuntu_packages
     else:
         print("Could not detect package manager type")
         sys.exit(-1)
 
-    for pck in packages:
+    for pck in all_packages:
         install_cmd = pck_mgr + " install -y " + pck
         print(install_cmd)
         execute_cmd(install_cmd)
 
-    update_pip = f"sudo -u {user} pip{python_version} install --upgrade pip"
+
+def pip_install_packages(user: str, python_version: str):
+    """
+    Install required python packages using pip
+
+    :param user:            User to install as
+    :type  user:            str
+    :param python_version:  Version of python to install on
+    :type  python_version:  str
+    """
+    packages = [
+        'wheel',
+        'deepdish',
+        'posix_ipc',
+        'inotify',
```

Comment:
I think `inotify` is only used for scheduler on cssdp - not sure if that changes things